python/tvm/driver/tvmc/runner.py

```diff
@@ -366,79 +464,102 @@ def run_module(
             "Try calling tvmc.compile on the model before running it."
         )
 
-    # Currently only two package formats are supported: "classic" and
-    # "mlf". The later can only be used for micro targets, i.e. with microTVM.
-    if tvmc_package.type == "mlf":
```

Comment:
can also do this in a follow-on. i think you still need to preserve this check when device != "micro"

Reply:
yeah, that seems correct. Ok, I'll prepare a follow-on PR for it. :+1:

Reply:
@areusch Actually, it's fixed in this PR, otherwise tests fail ;)